Presentation/UINavigationController+Presenting.swift

```diff
@@ -162,7 +162,14 @@ private extension UINavigationController {
 
 
 
-        if let coordinator = transitionCoordinator, !animated {
+        if #available(iOS 13.0, *), viewControllers.isEmpty {
+            // iOS 13 has an issue where the NavBar gets too narrow at initial presentation.
+            // Have tried different approaches to force UIKit to update the navbar layout on the current layout pass, but found no working solution.
```

Comment:
I'm wondering if it's connected to ```scrollEdgeAppearance```.

Reply:
for ex. :
```
    if #available(iOS 13.0, *), viewControllers.isEmpty {
        navigationBar?.scrollEdgeAppearance = .standardAppearance
    } else {
        ...
    }```

Reply:
I'll try it out!

Reply:
@TBernatskaya  Didn't do any difference. :-/